extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -116,12 +121,24 @@ export class AmpAdApiHandler {
         (waitForRenderStart.indexOf(this.baseInstance_.adType) >= 0);
 
     if (renderStartImplemented) {
+      const perf = performanceFor(this.baseInstance_.win);
+      const initTime = Date.now();
+      const adTag = adTypeAbbrevTable[this.baseInstance_.adType];
+      // Report performance for all renderStartImplemented cases.
+      // This is needed to calculate the perc of receiving nothing from ad.
+      perf.tickDelta(adTag, 0);
+      perf.flush();
```

Comment:
Do we need to flush immediately? Or can we just flush after we either get `render-start` or `no-content`?